demisto_sdk/commands/common/errors.py

```diff
@@ -444,6 +445,11 @@ def file_cannot_be_deleted(file_path: str):
     def suggest_fix(file_path: str, *args: Any, cmd: str = 'format') -> str:
         return f'To fix the problem, try running `demisto-sdk {cmd} -i {file_path} {" ".join(args)}`'
 
+    @staticmethod
+    @error_code_decorator
+    def empty_command_arguments(command_name):
+        return f"The arguments of the integration command `{command_name}` are None."
```

Comment:
```suggestion
        return f"The arguments of the integration command `{command_name}` must be defined (not None). For an empty list, use `arguments: []`."
```

or whatever the correct syntax for the command is